tests/test_addon_detail.py

```diff
@@ -442,3 +442,53 @@ def test_screenshot_keyboard_navigation(selenium, base_url, variables):
     assert '1' in addon.screenshots.screenshot_counter
     # send ESC to close the screenshot viewer
     addon.screenshots.esc_to_close_screenshot_viewer()
+
+
+@pytest.mark.nondestructive
+def test_add_to_collection_card(selenium, base_url, variables):
```

Comment:
In this tests I'm interested only in checking that the Add too Collection card is displayed on the detail page. Its functionality will be tested in detail when I'll be covering Collections as a whole, which will have a dedicated test suite.